controllers/__init__.py

```diff
@@ -46,11 +46,11 @@ def bootstrap_indexes(self) -> None:
         """Bootstrap indexes."""
         index_tuples = [
             ("flows", [("flow_id", pymongo.ASCENDING)], {"unique": True}),
+            ("flows", [("flow.cookie", pymongo.ASCENDING)], {}),
```

Comment:
Nicely done @Alopalao. Let's also keep `flow.cookie` in the compound index as well, that way queries for dpid and `cookie` when clients know which switches upfront they need, cases where just a single switch is needed are still handy to have.

Reply:
Ok, done.